samples/bootloader/src/main.c

```diff
@@ -58,13 +59,19 @@ static void boot_from(const struct fw_info *fw_info)
 		return;
 	}
 
+	if (fw_info->version > get_monotonic_counter()) {
+		err = set_monotonic_counter(fw_info->version);
+		printk("set_monotonic_counter() returned %d\n\r", err);
```

Comment:
This prints an error with the default configuration. This is not good IMO as it looks like something is actually failing.

Reply:
```suggestion
		if (num_monotonic_counter_slots() == 0) {
			printk("Monotonic HW version counter is disabled.\r\n");
		} else {
			printk("set_monotonic_counter() returned %d\n\r", err);
		}
```